README.md

```diff
@@ -1,34 +1,56 @@
 # cltdevs meetup manifesto
 
 ## Mission statement
-We grow Charlotte's tech community by bringing people together.
+We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating individual talents and organizations, and discover and build the missing elements of a thriving tech community.
 
-## Our target audience
-Developers for now (hence the name), but we welcome anyone that wants to join the community.
+## Who we are
+Our membership is primarily made up of people whose roles are involved in the tech industry and our partnerships include those whose livelihoods and businesses depend on it. Anyone involved directly or indirectly with tech of any kind or those who care about the tech community are invited to join our ranks.
 
-## How we grow Charlotte's tech community
-One word: **interactions**.
-
-The more interactions people create with each other, the more the sense of community will build.
+## How we accomplish our mission
+We believe that the talent we have in our city is a great asset and it should be carefully tended and grown. To accomplish this, we build bridges, we encourage networking and interaction, and we encourage and facilitate teaching, mentoring, and building. All the while, we strive for a diverse and egalitarian community and welcome anyone who desires to work towards this goal into our ranks.
```

Comment:
I think the title might be wrong here. This is where you take your mission and describe how your principles help you accomplish that. Perhaps, we're looking at consolidating some of the later stuff in here as a `Principles` section?